src/asynchronous/stream.rs

```diff
@@ -418,6 +418,9 @@ impl StreamSender {
             header,
             payload: buf,
         };
+
```

Comment:
Here's check is also redundant?

Reply:
We need check it before stream's sending